### PR TITLE
Added objects for benchmarking

### PR DESCRIPTION
**Pull request by atis06**

### CODE REVIEW

File: ondewo/nlu/agent.proto [hunk 1]
```diff
@@ -628,6 +638,33 @@ message ExportAgentResponse {
...
+
+    // Train-test split related parameters, for further info,
+    // check https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html
+    // Optional: bigger than zero
+    float test_size = 3;
+    // Optional: bigger than zero
+    float train_size = 4;
+    // Optional: random seed
+    int32 random_state = 5;
+    // Always True
```
Comment (gaborod16) [inline ondewo/nlu/agent.proto:659]: The field is ignored in the back-end since the stratified split always needs to shuffle.
Therefore, this actually cannot be configured.

File: ondewo/nlu/agent.proto [hunk 2]
```diff
@@ -628,6 +638,33 @@ message ExportAgentResponse {
...
+    bool shuffle = 6;
+}
+
+message ExportBenchmarkAgentResponse {
+    bytes agent_content = 1;
+    map<string, ListTrainingPhrasesResponse> training_phrases = 2;
```
Comment (gaborod16) [inline ondewo/nlu/agent.proto:665]: Can you add a description about what this training_phrases field is?
What is the "string key"? What does it contain? Intent ID? Intent name? Intent Display name?

File: ondewo/nlu/operation_metadata.proto [hunk 3]
```diff
@@ -89,6 +89,8 @@ message OperationMetadata {
         BUILD_AGENT_CACHE = 6;
         // train agent
         TRAIN_AGENT = 7;
+        // export benchmark agent to a zip file
+        EXPORT_BENCHMARK_AGENT = 8;
```
Comment (gaborod16) [inline ondewo/nlu/operation_metadata.proto:93]: +1

File: ondewo/nlu/agent.proto [hunk 4]
```diff
@@ -628,6 +638,33 @@ message ExportAgentResponse {
...
+    string parent = 1;
+    // Optional. The compression level of the zip file created.
+    // Valid range 1-9; the grpc-default value 0 will be mapped to 1,
+    // other values cause an error in the request validation handler.
+    int32 compression_level = 2;
+
+    // Train-test split related parameters, for further info,
+    // check https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html
+    // Optional: bigger than zero
+    float test_size = 3;
```
Comment (gaborod16) [inline ondewo/nlu/agent.proto:654]: if not passed, what is the default?

**Review by gaborod16 — CHANGES_REQUESTED** [review]
Looks good.
Just very small changes related to the understanding of the API :+1:

File: ondewo/nlu/agent.proto (hunk 4 above)
Comment (gaborod16) [reply ondewo/nlu/agent.proto:654]: @atis06 check this out

Comment (gaborod16) [reply ondewo/nlu/agent.proto:654]: I see, can you just mention the the default value comes from that API definition?

**Review by gaborod16 — APPROVED** [review]

File: ondewo/nlu/agent.proto (hunk 4 above)
Comment (gaborod16) [reply ondewo/nlu/agent.proto:654]: @atis06
